docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingFloatToString.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Conversions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingFloatToStringExample
+    {
+        /// <summary>
+        /// Helper class for retrieving the resulting data
+        /// </summary>
+        class SampleInfertDataWithInducedCategory
+        {
+            public float Age = 0;
+            public float Induced = 0.0f;
+            public string InducedCategory = default;
+        }
+
+        ///<summary>
+        /// This example demonstrates the use of floating types as the key type for ValueMappingEstimator by mapping a float-to-string value.
+        /// The mapping looks like the following:
+        /// <list>
+        ///     <item>1.0 -> Cat1</item>
+        ///     <item>2.0 -> Cat2</item>
+        /// </list>
+        /// </summary>
+        public static void Run()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Creating a list of keys based on the induced value from the dataset
+            // These lists are created by hand for the demonstration, but the ValueMappingEstimator does take an IEnumerable.
+            var inducedKeys = new List<float>()
+            {
+                1.0f,
+                2.0f
+            };
+
+            // Creating a list of values, these strings will map accordingly to each key.
+            var inducedValues = new List<string>()
+            {
+                "Cat1",
+                "Cat2"
+            };
+
+            // Constructs the ValueMappingEstimator making the ML.net pipeline
+            var pipeline = new ValueMappingEstimator<float, string>(ml, inducedKeys, inducedValues, ("InducedCategory", "Induced"));
```

Comment:
Can you call this from `mlContext`?

Reply:
This should be updated -- there are two cases that I can not call from mlContext. I have created issues for that.

---
In reply to: [252782089](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252782089) [](ancestors = 252782089)